app/javascript/react/components/ClueTile.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const ClueTile = (props) => {
+  return(
+    <>
+  <div>
+    < div className = "card qCard bg-secondary" >
+      <div className="card-body">
+        <p>{props.value}</p>
+      </div>
+      <div class="card-footer">
+      </div>
+    </div>
+  </div>
+    </>
+  )
```

Comment:
you have one more div than is needed, especially since you are using fragments with `<>`. remove